0-bootstrap/README.md

```diff
@@ -0,0 +1,57 @@
+# 0-bootstrap
+
+The purpose of this step is to bootstrap a GCP organization, creating all the required resources & permissions to start using the Cloud Foundation Toolkit (CFT). This step also configures Cloud Build & Cloud Source Repos for foundations code in subsequent stages.
+
+## Prerequisites
+
+1. A GCP [Organization](https://cloud.google.com/resource-manager/docs/creating-managing-organization)
+1. A GCP [Billing Account](https://cloud.google.com/billing/docs/how-to/manage-billing-account)
+1. Cloud Identity / Gsuite groups for organization and billing admins
+1. Membership in the `group_org_admins` group for user running terraform
+
+Further details of permissions required and resources created, can be found in the bootstrap module [documentation.](https://github.com/terraform-google-modules/terraform-google-bootstrap)
+
+## Usage
+
+1. Change into 0-bootstrap folder
+1. Rename terraform.example.tfvars to terraform.tfvars and update the file with values from your environment.
```

Comment:
We should include explicit instructions (ex. `cp terraform.example.tfvars terraform.tfvars`.

Reply:
Make sense, fixed in next push.